css-layout-classes/css-layout-classes/styles.css

```diff
@@ -0,0 +1,41 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: sans-serif;
+}
+
+.container {
+  max-width: 800px;
+  margin: 0;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  width: 100%;
+}
+
+.column-half {
+  width: 100%;
```

Comment:
each of these classes should have width values which correspond to their class name, for example `column-half` should have a width of 50%. Make sure to update all of the classes to the proper width values!